README.md

```diff
@@ -30,16 +29,25 @@ It provides cross-service call chain tracking and performance information collec
 ### Architecture Diagram
 ![image](./doc/images/EaseAgent-Architecture.jpg)
 
-## Features
+#### Description
+Use `Byte Buddy` to modify the bytecode, insert the code that needs to be run before and after the method is executed, refer to the following code usage: 
+```
+net.bytebuddy.asm.Advice.OnMethodEnter, net.bytebuddy.asm.Advice.OnMethodExit
+```
+1. When invoke the enhanced class, it will trigger the call of `Method Advice`
```

Comment:
You omitted the subject of the subordinate clause, which means the subject is the same as the main clause, but the main clause subject is `it`, What's the word `it` represented?  I think readers may be confused about this.